opentri/PSPBUILD

```diff
@@ -1,23 +1,17 @@
 # Maintainer: Wouter Wijsman <wwijsman@live.nl>
 pkgname=opentri
-pkgver=r6.ec3f441
+pkgver=r7.95b040d
 pkgrel=1
 pkgdesc="openTRI is a game engine for the PSP"
 arch=('mips')
 url="https://github.com/SamRH/openTRI"
 license=('GPL2')
-depends=('zlib' 'freetype2' 'libpng')
+depends=('zlib' 'freetype2')
 makedepends=()
 optdepends=()
-source=("git+https://github.com/SamRH/openTRI.git")
+source=("git+https://github.com/fjtrujy/openTRI.git")
 sha256sums=('SKIP')
 
-pkgver() {
```

Comment:
This is not optional unless the commit is set in source, which is preferable over using pkgver anyway, since we can't know if future versions will build with the same PSPBUILD and if they will work. If pkgver is used, we cannot guarantee that the build package will work.